addons/controllers/antrea/testcases/antrea-test-1.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: test-cluster-1
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: [ "192.168.0.0/16" ]
+    services:
+      cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: DockerCluster
+    name: test-cluster-1
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: DockerCluster
+metadata:
+  name: test-cluster-1
+---
+apiVersion: cni.tanzu.vmware.com/v1alpha1
```

Comment:
question: did y'all have to go to TAG to get this new group name reviewed? i am asking because i am wondering if i should go to TAG to get `auth.tanzu.vmware.com` reviewed...

cc @vijaykatam

Reply:
I am targeting Monday(2/7) for getting these reviewed